content/docs/hosting/BYOC.md

```diff
@@ -98,169 +99,178 @@ With Appbase.io Dashboard you can easily share cluster with your team members. E
-| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `ES_CLUSTER_URL`         | ElasticSearch Cluster URL                                                                                                                             |
-| `USERNAME`               | Username used for Basic Auth with ARC Cluster URL                                                                                                     |
-| `PASSWORD`               | Password used for Basic Auth with ARC Cluster URL                                                                                                     |
-| `JWT_RSA_PUBLIC_KEY_LOC` | When Role Based Access control is used for security. This Env helps in specifying the location of public key file that is used for verifying location |
-| `JWT_ROLE_KEY`           | Field used for getting the role from JWT token.                                                                                                       |
-| `ARC_ID`                 | Used for accessing the Arc Dashboard (This is to be set once the Arc Middleware is deployed and you have signed up for arc dashboard)                 |
+Now, let's add TLS certificate to EC2 public DNS / custom domain
 
-> **Note:** Make sure ElasticSearch cluster URL is open till the deployment is completed so that we can Verify ElasticSearch cluster.
+**Step 1-** Get TLS certificate with `.pem` & `.key` file
 
-#### **Getting Arc URL**
+> **Note:** You can obtain certificate using [https://letsencrypt.org/](https://letsencrypt.org/) or any other provider.
 
-Once the EC2 machine is created successfully, you can get Public DNS / IP address from the EC2 dashboard. This can be used with `USERNAME` and `PASSWORD` to access ElasticSearch data using Arc.
+**Step 2-** Access your EC2 instance using `ssh`. From EC2 dashboard you can select instance and click on **Connect** to get `ssh` login details about your machine.
 
-![](https://i.imgur.com/PDs8DK0.png)
+![EC2 Connect Screenshot](https://i.imgur.com/NRW1P6c.png)
 
-**Example**
+**Step 3-** Copy `.pem` file content to `/etc/pki/nginx/server.crt` using `vim`/`nano` editor
 
 ```bash
-curl -u="USERNAME:PASSWORD" http://EC2-PUBLIC-[IP/DNS]
+# Open the file
+sudo vim /etc/pki/nginx/server.crt
+# past the code
+cmd/ctrl + v
+# save the file
+:wq
 ```
 
-You can also point your domain to EC2 instance in order to access data using your domain. For more information on how to register/point domain to EC2 instance you can check [docs](https://aws.amazon.com/getting-started/tutorials/get-a-domain/).
+**Step 4-** Copy `.key` file content to`/etc/pki/nginx/private/server.key` using similar approach mentioned above.
 
-#### **Accessing Arc Dashboard**
+**Step 5-** Open `sudo vim /etc/nginx/nginx.conf`
```

Comment:
@lakhansamani Is `vim` installed on the AMI instance that we offer by default? Just making sure.

Reply:
Yes it is installed by default